lib/env.js

```diff
@@ -5,5 +5,6 @@ module.exports = {
   DEPLOYMENT_CONFIG_FILE: process.env.DEPLOYMENT_CONFIG_FILE || 'deployment-settings.yml',
   CREATE_PR_COMMENT: process.env.CREATE_PR_COMMENT || 'true',
   CREATE_ERROR_ISSUE: process.env.CREATE_ERROR_ISSUE || 'true',
-  BLOCK_REPO_RENAME_BY_HUMAN: process.env.BLOCK_REPO_RENAME_BY_HUMAN || 'false'
+  BLOCK_REPO_RENAME_BY_HUMAN: process.env.BLOCK_REPO_RENAME_BY_HUMAN || 'false',
+  FULL_SYNC_NOOP: process.env.FULL_SYNC_NOOP === 'true'
```

Comment:
@decyjphr 

We have different implementations for boolean environment variables. 
For example, `CREATE_PR_COMMENT` is handled as a string, while with this changes `FULL_SYNC_NOOP` is implemented as a boolean value. This is reflected in the different approaches in our unit tests.

I think it would be beneficial to standardize how we handle boolean environment variables across the project. Once we decide on the preferred implementation (string or boolean), I can create a new PR to refactor the existing code for consistency.
I can also update my changes to fit current implementation.